demo/src/test/java/com/example/test/core/resources/datas/datas.json

```diff
@@ -0,0 +1,9 @@
+[

```

Comment:
should change the name and write the new datadriven method return data Object

Reply:
Rename file datas.json to datasforstyleandcategory.json